Examples/php/pragmas/runme.php

```diff
@@ -2,4 +2,5 @@
 
 require "example.php";
 
+echo "Version - " . ((new ReflectionExtension('example'))->getVersion());
 ?>
```

Comment:
Did you get a chance to sort out the loose ends?  This is pretty close to being ready to merge.